### ISSUE TITLE
Fix relative path in the navigation

### ISSUE THREAD

**Issue by blefort** [dita4publishers/org.dita4publishers.html5#34]
`href` in the navigation use relative path in all situations, like '../concept/topicA.html'. 

My understanding is that it's not a big problem, it is not super clean implementation but it is a problem when clients have big documentation (over 100 Mo), because all topics in the same directory will have a longer href value which increase documentation size.

**Comment by blefort** [comment]
Done in severals steps, completed in 89404ac0846947a5b6dc87f8c42fe08f3d22334a

**Comment by drmacro** [comment]
The code is not working correctly. Reopenning this issue.

**Comment by blefort** [comment]
Could you give me an example ?

**Comment by drmacro** [comment]
The code was not calculating the correct relative path for some files.

The problem was trying to do everything with relative paths. It also wasn't adding the correct fragment ID for links to topics within to-content chunks.

There was also a bug in the base code that gets the unique topicrefs in that it wasn't handling the case where you have a to-content chunking topicref as a descendant of another to-content chunk--those topics weren't getting processed.

I reworked the code so the base navigation structure uses absolute paths, which makes calculating the relative paths easier and more reliable. And I fixed the chunk-handling bug.

**Comment by blefort** [comment]
Still have issue:

```
[xslt] /Users/juliesenecal/DITA-OT/plugins/org.dita4publishers.html5/xsl/map2html5Nav.xsl:161:13: Fatal Error! XPath syntax error at char 27 on line 161 in {...)) then}:
     [xslt]     Cannot find a matching 1-argument function named {http://dita2indesign.org/dita/functions}topicrefIsInChunk()
     [xslt] Failed to process null
```

I am working on it.

**Comment by blefort** [comment]
confirmed by travis
https://travis-ci.org/dita4publishers/dita4publishers/jobs/55087877

**Comment by blefort** [comment]
#37 is related

**Comment by blefort** [comment]
Proposed fix in 94da8d38aa6895b297f3643ba4e8bf65570d43d0. 
Need more test to confirm

**Comment by drmacro** [comment]
I updated the common.xslt module with this new function, so you need to have the latest version of that.

**Comment by blefort** [comment]
yes, I figured it but could not get the right path. I am doing tests at this moment. I like the way you use the resultUri variable to resolve the whole thing. There is a file strategy in your code that I have not tested yet. What is it for?
